tracing-forest/src/prelude.rs

```diff
@@ -0,0 +1,39 @@
+//! The module re-exports a number of useful components that you may wish
+//! to consume.
+
+pub use tracing::Instrument;
+
+pub use tracing::{
+    debug,
+    debug_span,
+    error,
+    error_span,
+    event,
+    info,
+    info_span,
+    span,
+    trace,
+    trace_span,
+    warn,
+    warn_span,
+};
+
+// As these macros are macro_export, they are published at the crate root.
+/*
+#[cfg(feature = "uuid")]
+pub use crate::{
+    uuid_debug_span,
+    uuid_error_span,
+    uuid_info_span,
+    uuid_span,
+    uuid_trace_span,
+    uuid_warn_span,
+};
+*/
+
+#[cfg(feature = "attributes")]
+pub use tracing;
+#[cfg(feature = "attributes")]
+pub use tracing::instrument;
+
+pub use tracing_subscriber::filter::LevelFilter;
```

Comment:
We should just have 5 method for each level to reduce the amount of exposed types. Thoughts?

Reply:
You mean "with_warn" or "with_error" etc? The problem I guess is that limits some flexibility in how you build the level from a configuration or similar ....